udp_driver/CMakeLists.txt

```diff
@@ -40,22 +40,19 @@ endif()
 include_directories(include)
 
 ament_auto_add_library(${PROJECT_NAME} SHARED
-  src/io_context.cpp
   src/udp_socket.cpp
   src/udp_driver.cpp
-  src/msg_converters/std_msgs.cpp
   examples/udp_driver_node.cpp)
 ament_target_dependencies(${PROJECT_NAME} "Boost")
 
 target_compile_options(${PROJECT_NAME} PUBLIC "-O0" PRIVATE -Wno-sign-conversion -Wno-conversion)
-target_link_libraries(${PROJECT_NAME} ${Boost_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} ${Boost_LIBRARIES} ${io_context_LIBRARIES})
```

Comment:
You can get rid of this because adding the dependency in `package.xml` means that everything added with `ament_auto_` macros automatically gets the include folders and linked against each dependency package.